src/scripts/experience/issue.js

```diff
@@ -428,5 +480,15 @@ define([
         }
     };
 
+    Issue.prototype.focus = function() {
+        this.elm.addChild(this._square);
```

Comment:
Since PIXI.js has a tree of graphics buffers that it composites into the final image to draw, we just draw the square once and then add/remove it from the tree to display or hide it.